src/constructs/iam/policies/base-policy.ts

```diff
@@ -0,0 +1,20 @@
+import type { CfnPolicy, PolicyProps } from "@aws-cdk/aws-iam";
+import { Policy } from "@aws-cdk/aws-iam";
+import type { GuStack } from "../../core";
+
+export interface GuPolicyProps extends PolicyProps {
+  overrideId?: boolean;
+}
+
+export class GuPolicy extends Policy {
+  static defaultProps: Partial<GuPolicyProps> = {};
```

Comment:
Is it worth keeping this if it doesn't do anything at the moment?

Reply:
Yes, but not for for this PR to keep it as no-op as possible. I've got another branch that makes a few changes to this area, so might make sense to add it there?